contributed_definitions/nyaml/NXion.yaml

```diff
@@ -26,45 +26,40 @@ NXion(NXobject):
     unit: NX_UNITLESS
   isotope_vector(NX_UINT):
     doc: |
-      A vector of isotope hash values. These values have to be stored in an array,
-      sorted in decreasing order. The array is filled with zero hash values
-      indicating unused places. The individual hash values are built with the
-      following hash function:
+      Row vector of isotope hash values. The individual hash values are
+      built with the following hash function:
       
       The hash value :math:`H` is :math:`H = Z + N*256` with :math:`Z`
       the number of protons and :math:`N` the number of neutrons
-      of each isotope respectively.
+      of each isotope respectively. Z and N have to be 8-bit unsigned integers.
       
-      Z and N have to be 8-bit unsigned integers.
+      The array is sorted in decreasing order with unused values nullified.
       For the rationale behind this `M. Kühbach et al. (2021) <https://doi.org/10.1017/S1431927621012241>`_
     unit: NX_UNITLESS
     dim: (1, n_ivecmax)
   nuclid_list(NX_UINT):
     doc: |
-      A supplementary row vector which decodes the isotope_vector into
-      a human-readable matrix of nuclids with the following formatting:
-      
-      The first row specifies the isotope mass number, i.e. using the hashvalues
-      from the isotope_vector this is :math:`Z + N`. As an example for a
-      carbon-14 isotope the number is 14.
-      The second row specifies the number of protons :math:`Z`, e.g. 6 for the
-      carbon-14 example. This row matrix is thus mapping the notation of
-      using superscribed isotope mass and subscripted number of protons
-      to identify isotopes.
-      Unused places filling up to n_ivecmax need to be filled with zero.
+      Table which decodes the isotope_vector into a human-readable matrix of nuclids.
+      The first row specifies the nuclid mass number, i.e. using the hashvalues
+      from the isotope_vector this is :math:`Z + N`. 
+      Taking a carbon-14 nuclid as an example the mass number is 14.
+      The second row specifies the number of protons :math:`Z`, e.g. 6 as
+      for the carbon-14 example. 
+      This notation is thus mapping the notation of using superscribed
+      nuclid mass and subscripted number of protons. The array is stored
+      matching the order of isotope_vector with unused values nullified.
     unit: NX_UNITLESS
     dim: (2, n_ivecmax)
-  color(NX_CHAR):
-    doc: |
-      Color code used for visualizing such ions.
+  # color(NX_CHAR):
```

Comment:
Any reason why this is removed?

Reply:
Color used for visualization heavily depends on the consuming application typically the proprietary GUI application used for analyzing atom probe data, although possibly significant in these software packages, the colors are irrelevant for NOMAD as currently the atoms in 3D are rendered if at all using north and then using different color schemes, as such, the field has never been used and thus has been removed for now

Reply:
Ok, fine with me